pom.xml

```diff
@@ -171,6 +171,7 @@
           <dependencySourceExcludes>
             <dependencySourceExclude>com.hms_networks.americas.sc.mvnlibs:ewon-etk:*</dependencySourceExclude>
           </dependencySourceExcludes>
+          <failOnWarnings>true</failOnWarnings>
```

Comment:
This should be carried upstream to the starter project.

Reply:
Yeah I agree